src/RA_Dlg_MemBookmark.cpp

```diff
@@ -550,7 +550,7 @@ void Dlg_MemBookmark::SetupColumns(HWND hList)
             col.iSubItem   = idx
         };
 
-        if (idx == (ra::COLUMN_TITLE.size() - 1))
+        if (idx == (ra::to_signed(ra::COLUMN_TITLE.size()) - 1))
```

Comment:
Came up as warning. Level 4 warnings weren't enabled when I originally did this.

Reply:
I saw this too. My fix was to make `idx` unsigned: https://github.com/RetroAchievements/RAIntegration/pull/118/files#diff-ec82be412bc671652491c73da885f532R539

Reply:
I made it signed because of lv_columns  members being signed.